src/include/dhandle.h

```diff
@@ -69,7 +69,10 @@ struct __wt_data_handle {
     const char *checkpoint; /* Checkpoint name (or NULL) */
     const char **cfg;       /* Configuration information */
     const char *meta_base;  /* Base metadata configuration */
-
+#ifdef HAVE_DIAGNOSTIC
+    const char *orig_meta_base; /* Copy of the base metadata configuration */
+#endif
+    size_t orig_meta_base_length; /* Base metadata length */
```

Comment:
Since this length field is really for `meta_base` move it above the `HAVE_DIAGNOSTIC` to be next to `meta_base`.